sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/CHANGELOG.md

```diff
@@ -2,3 +2,6 @@
 
 
 ## 1.0.0 (Unreleased)
+
+### Breaking Changes
```

Comment:
We usually don't need to update the changelog for sample projects because we're not going to release them.

Reply:
ok